### PR TITLE
Expose ::android::view::WindowInsets_Type

### PR DESCRIPTION
**Pull request by todi1856**
Need it for this code
```
            bool decorFitsSystemWindows = !(fullscreen || s_RenderOutsideSafeArea);
            activity.GetWindow().SetDecorFitsSystemWindows(decorFitsSystemWindows);

            android::view::WindowInsetsController controller = activity.GetWindow().GetInsetsController();
            if (controller)
            {
                static int s_Insets = android::view::WindowInsets_Type::StatusBars() |
                    android::view::WindowInsets_Type::NavigationBars();

                if (decorFitsSystemWindows)
                {
                    controller.Show(s_Insets);
                }
                else
                {
                    controller.Hide(s_Insets);
                    controller.SetSystemBarsBehavior(android::view::WindowInsetsController::fBEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE());
                }
            }
```

### CODE REVIEW

File: JniBridge.bee.cs [hunk 1]
```diff
@@ -91,6 +91,7 @@ static class Platform
         "::android::view::Display",
         "::android::view::Gravity",
         "::android::view::SurfaceView",
+        "::android::view::WindowInsets_Type",
```
Comment (aurimasc) [inline JniBridge.bee.cs:94]: Looks like some of these use underscore and some (MediaCode::BufferInfo) a double colon. Is there a difference?
I'd prefer to have WindowInsets::Type, a more direct counterpart.

Comment (todi1856) [reply JniBridge.bee.cs:94]: "::android::view::WindowInsets::Type" doesn't generate the API

Comment (aurimasc) [reply JniBridge.bee.cs:94]: But what's the difference from https://developer.android.com/reference/android/media/MediaCodec.BufferInfo
Or perhaps we don't have API generated for BufferInfo?

Comment (todi1856) [reply JniBridge.bee.cs:94]: we do have api generated for BufferInfo, not sure - what's difference in this case. This is something we would need investigate, but obviously not now... If you want I can fill the ticket

Comment (aurimasc) [reply JniBridge.bee.cs:94]: > If you want I can fill the ticket

Please do.

**Review by aurimasc — APPROVED** [review]

File: JniBridge.bee.cs (hunk 1 above)
Comment (todi1856) [reply JniBridge.bee.cs:94]: Btw, I noticed if you remove "::android::media::MediaCodec::BufferInfo", the binding is still generated, thus maybe it's like you said - dependencies are generated, so "::android::media::MediaCodec_BufferInfo" is correct, and "::android::media::MediaCodec::BufferInfo" is incorrect.

Do you still want a ticket?

Comment (aurimasc) [reply JniBridge.bee.cs:94]: So what is the generated class name?
If it's MediaCodec_BufferInfo, then it's all by design.

Comment (todi1856) [reply JniBridge.bee.cs:94]: In C++ it's android::media::MediaCodec_BufferInfo

**Pull request merged**